.github/workflows/deploy.yml

```diff
@@ -0,0 +1,35 @@
+name: GH Page Deploy
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This action will only run when the PR is merged into the `main` branch. Please adjust this PR to merge to the `main` branch.